models/ctiStatus.ts

```diff
@@ -20,6 +21,10 @@ export const ctiStatus = createModel<RootModel>()({
       state.isPhoneRinging = isPhoneRinging
       return state
     },
+    setIdIntervalStatus: (state, idInterval: any) => {
```

Comment:
```suggestion
    setIdInterval: (state, idInterval: any) => {
```
Let's keep reducer names consistent